<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QWidget" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>210</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lucida Console</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>config</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/icon/mitsuyu/img/grpc.jpg</normaloff>:/icon/mitsuyu/img/grpc.jpg</iconset>
  </property>
  <widget class="QLineEdit" name="edit_addr">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>www.example.com:443</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_add">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sname">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>service</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_sname">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>mitsuyux</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_sni">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>www.example.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sni">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>sni</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_verify">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>verify</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_compress">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>compress</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rst">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Console</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
